app/views/compute_resources/form/_ovirt.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= text_f f, :url, :size => "col-md-8", :help_block => _("e.g. https://ovirt.example.com/api"), :help_inline => documentation_button('5.2.7oVirt/RHEVNotes') %>
 <%= text_f f, :user, :help_block => _("e.g. admin@internal") %>
 <%= password_f f, :password, :unset => unset_password? %>
+<%= checkbox_f f, :filtered_api, :help_block => _("If you don't have admin permissions you can use slower filtered API, that works with user account."), :label => _('Filtered API') %>
```

Comment:
I don't think this help block is very helpful, a quick DuckDuckGo search doesn't say nearly anything about 'filtered API', I could only find 1 or 2 references about setting the header 'Filter=true'. 

Expecting users to look for that kind of info that is a bit too much, how about calling the option 'User level API' and just mention in the help block to enable it if the user is not an oVirt administrator?